docs/releases/v1.247349.0.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+## [v1.247351.0](https://github.com/aws/amazon-cloudwatch-agent/tree/v1.247351.0) (2022-05-12)
```

Comment:
Why is the name of this file v1.247349.0, but the release tag here v351?

Reply:
My bad. Fixed it in the newest commit. Moreover, avoid merging this one until the story's grooming is done though.